components/eam/src/physics/cam/micro_p3.F90

```diff
@@ -815,7 +815,7 @@ SUBROUTINE p3_main_part2(kts, kte, kbot, ktop, kdir, do_predict_nc, do_prescribe
 
       call prevent_ice_overdepletion(pres(k), t_atm(k), qv(k), latent_heat_sublim(k), inv_dt, qidep, qi2qv_sublim_tend)
 
-      call ice_supersat_conservation(qidep,qinuc,cld_frac_i(k),qv(k),qv_sat_i(k),latent_heat_sublim(k),th_atm(k)/exner(k),dt, &
+      call ice_supersat_conservation(qidep,qinuc,cld_frac_i(k),qv(k),qv_sat_i(k),latent_heat_sublim(k),th_atm(k)/inv_exner(k),dt, &
            qi2qv_sublim_tend, qr2qv_evap_tend)
```

Comment:
prob better to multiply by exner instead of dividing by inv_exner? ;-) . Though maybe you're trying to keep things BFB?

Reply:
Agreed.  I can test to make sure switching to `*exner` is still BFB.  I think it would definitely make the code make more sense.